pkg/cmd/server/start/start_master.go

```diff
@@ -604,6 +609,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunPersistentVolumeClaimBinder(binderClient)
 		kc.RunPersistentVolumeProvisioner(provisionerClient)
 		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
+		kc.RunGCController(gcClient)
```

Comment:
upstream starts this right after the replication controller. @deads2k I don't think it matters, but can you think of a reason we should match that?